src/app/globals.css

```diff
@@ -1,3 +1,5 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700;800;900&family=Red+Hat+Text:wght@500;700&display=swap');
```

Comment:
Why is the font being imported here? Inter is a variable font. You can use tailwind classes instead. Please remove